main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/ONSdigital/dp-healthcheck/healthcheck"
+	"github.com/ONSdigital/dp-table-renderer/api"
+	"github.com/ONSdigital/dp-table-renderer/config"
+	"github.com/ONSdigital/log.go/log"
+)
+
+var (
+	// BuildTime represents the time in which the service was built
+	BuildTime string
+	// GitCommit represents the commit (SHA-1) hash of the service that is running
+	GitCommit string
+	// Version represents the version of the service that is running
+	Version string
+)
+
+func main() {
```

Comment:
I think it would be worth splitting this function in two so that it is consistent with other repositories: 

```
func main() {
	log.Namespace = serviceName
	ctx := context.Background()

	if err := run(ctx); err != nil {
		log.Event(ctx, "fatal runtime error", log.Error(err), log.FATAL)
		os.Exit(1)
	}
}

func run(ctx context.Context) error {
	signals := make(chan os.Signal, 1)
...
```
And then the run function would return errors instead of os.Exit(1)

Reply:
I've looked at splitting this now. It looks to run as before, but if you find anything that could be cleaned up or optimised, do let me know